[azure-devops-exporter] Add ability to export both automated and manual tests (#1865)
vividus-framework/vividus: docs/modules/integrations/pages/azure-devops-exporter.adoc

== Azure DevOps Export Properties

[cols="2,1,1,3", options="header"]
|===

> |Property

Reviewer: @valfirst  does it make sense to follow common rules for properties definition across all the docs and made required props as a bold

Reviewer 2: yes, it makes sense

== Azure DevOps Export Properties

NOTE: The properties marked with *bold* are mandatory.

[cols="2,1,1,3", options="header"]
|===

|Property
|Default value
|Acceptable values
|Description

|`*azure-devops-exporter.json-results-directory*`
|`empty`
|string
a|Path to directory with test execution JSON results.

The path is managed by the `bdd.report-directory` property which default value is `output/reports/jbehave` relatively to the tests directory.

Please make sure that the `bdd.configuration.formats` property includes JSON value in order to generate test execution results.

|`*azure-devops-exporter.organization*`
|`empty`
|string
|The name of the Azure DevOps organization.

|`*azure-devops-exporter.project*`
|`empty`
|string
|Project ID or project name.

|`*azure-devops-exporter.area*`
|`empty`
|string

> |The area of the product with which a test case is associated.

Reviewer 2: ```suggestion
|The area of the product which a test case is associated with.
```

Author: copied from official docs :)

> |`*azure-devops-exporter.section-mapping.steps*`

Reviewer 2: it shouldn't be mandatory, if it has a default value

|`azure-devops-exporter.organization`
|true
|string
|The name of the Azure DevOps organization.

|`azure-devops-exporter.project`
|true
|string
|Project ID or project name

|`azure-devops-exporter.area`
|true
|string
|The area of the product with which a test case is associated

|`azure-devops-exporter.section-mapping.steps`

> |AUTOMATED

Reviewer: This is required or not required field?

|AUTOMATED or MANUAL

> |The type of test cases that will be exported into `Steps` section of `Test Case` work item, for the remaining test case type the `Summary` section will be used instead 

Reviewer 2: ```suggestion
|The scenario part to export into `Steps` section of `Test Case` work item, the remaining scenario part is exported into `Summary` section.
```
